pyproject.toml

```diff
@@ -0,0 +1,117 @@
+# PEP-517 build
+
+[build-system]
+requires = ["poetry_core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+# Poetry tooling configuration
+
+[tool.poetry]
+name = "josepy"
+version = "1.14.0.dev0"
+description = "JOSE protocol implementation in Python"
+license = "Apache License 2.0"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Topic :: Internet :: WWW/HTTP", 
+    "Topic :: Security",
+]
+homepage = "https://github.com/certbot/josepy"
+authors = ["Certbot Project <certbot-dev@eff.org>"]
+readme = "README.rst"
+include = [
+    "CHANGELOG.rst",
+    "CONTRIBUTING.md",
+    ".coveragerc", "pytest.ini", "tox.ini",
```

Comment:
`.coveragerc` and `pytest.ini` no longer exist and while this is arguably out of scope of this PR, I see no reason for us to continue to include `tox.ini`.
```suggestion
```